lua/luasnip/extras/filetype_functions.lua

```diff
@@ -1,27 +1,34 @@
+local function fts_from_ts_lang(lang)
+	-- In case of someone using nvim <= 0.9
+	if vim.treesitter.language and vim.treesitter.language.get_filetypes then
+		return vim.treesitter.language.get_filetypes(lang)
+	end
+	return { lang }
```

Comment:
I _think_ we should also return `lang` in the first case, there are probably users who have their latex-snippets as latex due to the current behaviour, and those would no longer be loaded like this.

Also, I hope that this does not cause issues where a filetype is loaded when it shouldn't be (manually disassociating them is not really possible), but since it's a better default, let's wait with making this configurable until there is a concrete issue-case.

Reply:
I will added `lang` to the `fts` if it's not there already (most of the time it will be).

Reply:
Sounds good :ok_hand: